### PR TITLE
Update Cypress commands

### PR DESCRIPTION
**Pull request by anselmbradford**
https://github.com/cfpb/consumerfinance.gov/pull/8459 added fast cypress script.
https://github.com/cfpb/consumerfinance.gov/pull/8518 added a `cypress open` script.

The usual yarn way of opening Cypress is running `yarn cypress open`, however, I noticed in the docs that the recommended [best practice](https://docs.cypress.io/guides/getting-started/opening-the-app) is to not name the script `cypress`, and instead show `cy` as an example. 

This PR unifies the Cypress scripts together in the `/scripts/` directory and creates a single `cy` script, which takes `fast` or `open` arguments, or otherwise runs as `run`.

## Changes

- Add Cypress scripts to `/scripts/`
- Update Cypress command to `cy`.

## How to test this PR

1. `yarn cy` runs Cypress headless.
2. `yarn cy open` runs interactive Cypress app.
3. `yarn cy fast` runs changed specs only Cypress in headless mode.

### CODE REVIEW

File: scripts/cypress.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if [ ${1:-run} = "fast" ]; then
```
Comment (wpears) [inline scripts/cypress.sh:3]:
```suggestion
if [ ${1:-run} = "fast" ]; then
```
if [ "$1" = "fast"]; then

File: scripts/cypress.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    ./scripts/cypress-fast-specs.sh && ./node_modules/.bin/cypress run
+elif [ ${1:-run} = "open" ]; then
```
Comment (wpears) [inline scripts/cypress.sh:5]:
```suggestion
elif [ "$1" = "open" ]; then
```

same as above

Comment (anselmbradford) [reply scripts/cypress.sh:5]: IIRC this is needed for cases where there is no argument. I.e. `yarn cy`.

File: scripts/cypress.sh (hunk 1 above)
Comment (anselmbradford) [reply scripts/cypress.sh:3]: (Noted below, but checked it here). Without the default it throws ` [: =: unary operator expected` on a command without an argument (`yarn cy`).

Comment (wpears) [reply scripts/cypress.sh:3]: That was true even with the quoted `"$1"`?

Comment (anselmbradford) [reply scripts/cypress.sh:3]: Ahhhhh no I missed the quotes! Thank you! These changes look good with the quotes.

**Review by wpears — APPROVED** [review]
Cool!

**Pull request merged**